sdk/core/azure-core/azure/core/pipeline/base.py

```diff
@@ -60,7 +60,7 @@ def send(self, request):
 class _TransportRunner(HTTPPolicy):
 
     def __init__(self, sender):
-        # type: (HttpTransport) -> None
+        # type: (Any) -> None
```

Comment:
Is sender any?

Reply:
We had HTTPSender in msrest which we don't have anymore. Can't find a relevant sender class.

Reply:
I think HttpTransport is correct